.github/workflows/build.yml

```diff
@@ -0,0 +1,22 @@
+name: Build and deploy
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          submodules: 'recursive'
+
+      - name: Build
+        run: |
+          make modules -j`nproc`
```

Comment:
I was looking through the other PR's and #48 is suggesting that -j`nproc` be replaced with -j$(nproc)

Can this be done here as well?

Reply:
sure, done in ec11aac